cirq/ops/pauli_string.py

```diff
@@ -57,8 +58,8 @@ def equal_up_to_sign(self, other: 'PauliString') -> bool:
     def __getitem__(self, key: raw_types.QubitId) -> Pauli:
         return self._qubit_pauli_map[key]
 
-    def get(self, key: raw_types.QubitId, default: Optional[Pauli] = None
-            ) -> Optional[Pauli]:
+    def get(self, key: raw_types.QubitId, default: Union[Pauli, TDefault] = None
```

Comment:
Just `default: TDefault`, not a union. `TDefault` already includes `Pauli` as a possible option.